runelite-client/src/test/java/net/runelite/client/util/ColorUtilTest.java

```diff
@@ -123,8 +126,66 @@ public void colorLerp()
 	public void colorToHexCode()
 	{
 		COLOR_HEXSTRING_MAP.forEach((color, hex) ->
-		{
-			assertEquals(hex, ColorUtil.colorToHexCode(color));
-		});
+			assertEquals(hex, ColorUtil.colorToHexCode(color)));
+	}
+
+	@Test
+	public void colorToAlphaHexCode()
+	{
+		COLOR_ALPHA_HEXSTRING_MAP.forEach((color, hex) ->
+			assertEquals(hex, ColorUtil.colorToAlphaHexCode(color)));
+	}
+
+	@Test
+	public void isHex()
+	{
+		COLOR_HEXSTRING_MAP.forEach((_color, hex) ->
+			assertTrue(ColorUtil.isHex(hex)));
+	}
+
+	@Test
+	public void isAlphaHex()
+	{
+		COLOR_ALPHA_HEXSTRING_MAP.forEach((_color, hex) ->
+			assertTrue(ColorUtil.isAlphaHex(hex)));
+	}
+
+	@Test
+	public void fromString()
+	{
+		String WHITE_MAX_ALPHA = "-1";
+		String WHITE_ZERO_ALPHA = "0xffffff";
+		String TOO_LARGE = "0xffffffff";
+		String INVALID_FORMAT = "ffffff";
+
+		assertEquals(Color.WHITE, ColorUtil.fromString(WHITE_MAX_ALPHA));
+		assertEquals(new Color(0xffffff, true), ColorUtil.fromString(WHITE_ZERO_ALPHA));
```

Comment:
The expected value is much clearer to read if it is constructed instead as `ColorUtil.colorWithAlpha(Color.WHITE, 0)`

Reply:
That didn't occur to me! I've done that now.